src/borg/testsuite/archiver.py

```diff
@@ -2551,6 +2551,16 @@ def test_recreate_recompress(self):
         assert int(csize) < int(size)
         assert sha256_before == sha256_after
 
+    def test_recreate_timestamp(self):
+        self.create_test_files()
+        self.cmd('init', '--encryption=repokey', self.repository_location)
+        archive = self.repository_location + '::test0'
+        self.cmd('create', archive, 'input')
+        self.cmd('recreate', '--timestamp', "1970-01-01T00:00:00", '--comment',
+                 'test', archive)
+        info = self.cmd('info', archive).splitlines()
+        assert any([re.search(r'Time \(start\).+ 1970-01-01', item) for item in info])
```

Comment:
in case that timestamp is considered local time, please use something later to avoid an underflow in utc.

Reply:
Working on it to have a cleaner test.
Actually, the timestamp is considered localtime after some checks (I'm UTC+1 and I can see
`Time start: 1970-01-01 01:00:00`.
The code is nearly ready, tests are running locally to be sure.

Reply:
I'ld just get at least 24h away from time 0.